mix.exs

```diff
@@ -76,7 +79,11 @@ defmodule Franklin.MixProject do
       {:phx_live_storybook, "~> 0.4.0"},
 
       # To help generate some fake test data
-      {:faker, "~> 0.17", only: :test}
+      {:faker, "~> 0.17", only: :test},
+
+      # Static analysis
+      {:credo, "~> 1.6", only: [:dev, :test], runtime: false},
+      {:dialyxir, "~> 1.0", only: [:dev], runtime: false}
```

Comment:
```suggestion
      {:dialyxir, "~> 1.0", only: [:dev, :test], runtime: false}
```

I think this needs to be available in the `:test` config for GitHub Actions to see it.

Reply:
To be clear, this change was made in response to the CI output:

`** (Mix) The task "dialyzer" could not be found`

Upon adding `:test` we now get new `dialyzer` failures.

I think the ideal way forward is to revert this addition of `:test` and then to break out credo and dialyzer as distinct steps. For dialyzer make sure to run it as `MIX_ENV=dev`.

Happy to do this, but figured I post first and give you time to offer feedback/respond.

Reply:
Actually, if we run `dialyzer` with `MIX_ENV=dev` the entire project will have to recompile because the compilation was run using `MIX_ENV=test` at the top of the CI config.

I think keeping the `dialyzer` dependency for `:test` is fine in this case?